cmd/ci-operator-config-mirror/main.go

```diff
@@ -109,6 +111,10 @@ func main() {
 			return nil
 		}
 
+		if len(o.onlyOrg) > 0 && repoInfo.Org != o.onlyOrg {
```

Comment:
nit: you don't need the `len()` check and we should comment that we're skipping like in the next `if`

Reply:
We need the len check to make sure that the flag is specified. Otherwise, we will skip all configs. +1 for the log, I will address this.

Reply:
Err, sure. Could also flip the logic.